Configurations/unix-Makefile.tmpl

```diff
@@ -674,6 +674,12 @@ update: generate errors ordinals
 generate: generate_apps generate_crypto_bn generate_crypto_objects \
           generate_crypto_conf generate_crypto_asn1
 
+doc-nits:
+	$(PERL) util/find-doc-nits.pl -n >doc-nits
+	if [ ! -s doc-nits ] ; then exit 0 ; fi
```

Comment:
Have you tested that this works as intended?  I have some doubts, and am thinking you'll need to change this and the following two lines with:

``` make
	if [ -s doc-nits ] ; then cat doc-nits; exit 1; fi
```